tekton/README.md

```diff
@@ -18,13 +18,13 @@ GENERIC_SECRET=release-secret
 SERVICE_ACCOUNT=default 
 GCP_ACCOUNT="release-right-meow@tekton-releases.iam.gserviceaccount.com"
 
-# 1. Create a private key for the service account, which you can use
+# 1. Create a GCP private key for the service account. You may want to keep a copy somewhere safe. 
 gcloud iam service-accounts keys create --iam-account $GCP_ACCOUNT $KEY_FILE
 
-# 2. Create kubernetes secret, which we will use via a service account and directly mounting
+# 2. Sore GCP key in a secret
 kubectl create secret generic $GENERIC_SECRET --from-file=./$KEY_FILE
 
-# 3. Add the docker secret to the service account
+# 3. Patch the GCP key onto the service account to be used to run the release pipeline. 
 kubectl patch serviceaccount $ACCOUNT -p "{\"secrets\": [{\"name\": \"$GENERIC_SECRET\"}]}"
```

Comment:
Should be SERVICE_ACCOUNT and there's an of that should be an off later, otherwise lgtm